INSTALL.rst

```diff
@@ -308,13 +308,16 @@ You should also set apache to start on boot::
 
   sudo chkconfig httpd on
 
-The networking server may be started by running::
+The networking server may be started by running the following as the HaaS user::
 
   haas serve_networks &
```

Comment:
If we're adding a proper init script, this should probably become `service haas-network start` (and will need to be run as root, for the usual reason).